agency/README.md

```diff
@@ -176,28 +180,22 @@ Query Parameters:
 | ----- | ---- | ----------------- | ----- |
 | `provider_id` | UUID | Required | A UUID for the Provider, unique within MDS |
-| `service_area_id` | UUID  | Optional | If provided, retrieve a specific service area (e.g. a retired or old service area). If omitted, will return all active service areas. |
 
-Response:
+Response: 
+
+The response is an order  JSON / List with each item has the following members. 
 
 | Field | Types  | Required/Optional | Other |
 | ----- | ---- | ----------------- | ----- |
 | `service_area_id` | UUID | Required |  |
-| `service_start_date` | Unix Timestamp | Required | Date at which this service area became effective |
-| `service_end_date` | Unix Timestamp | Required | Date at which this service area was replaced. If currently effective, place NaN |
-| `service_area` | MultiPolygon | Required | |
-| `prior_service_area` | UUID | Optional | If exists, the UUID of the prior service area. |
-| `replacement_service_area` | UUID | Optional | If exists, the UUID of the service area that replaced this one |
-| `type` | Enum | Required |  See [area types](#area-types) table |
-
-### Area types
-
-| `type` | Description |
-|------------| ----------- |
-| unrestricted | Areas where devices may be picked up/dropped off. A provider's unrestricted area shall be contained completely inside the agency's unrestricted area for the provider in question, but it need not cover the entire agency unrestricted area. See the provider version of the service areas endpoint |
-| restricted | Areas where device pick-up/drop-off is not allowed |
-| preferred_pick_up | Areas where users are encouraged to pick up devices |
-| preferred_drop_off | Areas where users are encouraged to drop off devices |
+| `service_start_date` | Unix Timestamp | Required | Datetime at which this service area enters into effect. For the base service_area, the time should be the start of permitted operations in the City. |
+| `service_end_date` | Unix Timestamp | Optional | Datetime at which the service area is no longer in effect. For example, a temporary restriction for an event can be placed in advance and set to end at a certain time. |
+| `service_area` | MultiPolygon | Required | Per [GeoJSON spec](https://tools.ietf.org/html/rfc7946), must follow the right-hand rule to designate interior.   |
+| `allowed_events` | List | Required | List of permitted events inside the area | 
+| `forbidden_events` | List | Required | List of events that is not permitted inside the area | 
+| `speed_cap` | Integer | Optional | Speed cap to be enable on device, in integers, for devices that support governors. Expressed in miles per hour |
+| `preferred_pick_up` | Boolean | Optional | If true, means this service area is an incentive zone for user pick up, and should be expressed to users |
+| `preferred_drop_off` | Boolean | Optional |  If true, means this service area is an incentive zone for user drop offs, and should be expressed to users in app |
 
```

Comment:
Does it make sense to refactor those last three columns `speed_cap`, `preferred_pick_up`, `preferred_drop_off` into an enum or sub-object? It feels like there might be more of these "attributes" in the future - for example a field that mirrors the `allowed_events` as a list of enum types. 

I haven't thought about this too hard yet. Just throwing it out there.

Reply:
hmmm yeah. I think speed_cap is worth separating because `type(int)`, but how does something like: 

`zones: [list_of_zone_types]`

with `zone_types` == preferred_pick_up, preferred_drop_off, valet_station, any others?

Reply:
👍 I like `zone_type`. Also agreed that `speed_cap` is different enough.

One clarification on `speed_cap` - this is intended to be the new maximum speed the device is supposed to be able to go, within this zone - right? We should probably be crystal clear in the definition column there.